classes/admin_settings_aws_region.php

```diff
@@ -61,16 +61,35 @@ public function output_html($data, $query='') {
             }
         }
 
-        $context = (object) [
-            'size' => $this->size,
-            'id' => $this->get_id(),
-            'name' => $this->get_full_name(),
-            'value' => $data,
-            'forceltr' => $this->get_force_ltr(),
-            'options' => $options,
-        ];
-
-        $element = $OUTPUT->render_from_template('local_aws/setting_list', $context);
+        if (!empty($CFG->branch) && $CFG->branch > 31) {
+            $context = (object) [
+                'size' => $this->size,
+                'id' => $this->get_id(),
+                'name' => $this->get_full_name(),
+                'value' => $data,
+                'forceltr' => $this->get_force_ltr(),
+                'options' => $options,
+            ];
+            $element = $OUTPUT->render_from_template('local_aws/setting_list', $context);
```

Comment:
as discussed, not much point in supporting two code paths that do the same thing, lets just ditch the mustache template and do it the html_writer way for everyone

Reply:
Done